### ISSUE TITLE
test error in market

### ISSUE THREAD

**Issue by chegnyanjun** [YHJRUBY/ReID_baseline#1]
Hello,  I have used your code to reproduce the reuslt,but the result was not as good as your said .My training loss nearly the same as your training loss;And i just changing one place(utils.py 34 row) for where has a problem,Besides, I half the batch for my gpu is not big enough .That's all.The last test result as below:
top1:0.332542
top5:0.598827
top10:0.713183
mAP:0.158654
And i didn't know the reason what makes the result so bad,If you can give me some suggestions,I will be very happy.

**Comment by ye-hongjiang** [comment]
@chegnyanjun If you obtained the same training loss as mine, I think the problem exists in testing phase. Are you sure that you load the right model that was trained on the same dataset? BTW, what did you change on `utils.py row 34`? I think there's no problem on this line.

**Comment by chegnyanjun** [comment]
Thank you very much.I have solved the problem by myself.because i used pytorch0.3,So i changed the utils.py row 34(i deleted  map_location='cpu').After i changed the pytorch from 0.3 to 0.3.1,the problem solved,and the test result very well.Thank you very much for you can answer my question,I am very happy.

**Comment by ye-hongjiang** [comment]
@chegnyanjun Thanks for your attention. Looking forward to your star if you love my job :)
